CHANGELOG.md

```diff
@@ -20,6 +20,7 @@
 * Add `BeEql` cop which looks for expectations that can use `be(...)` instead of `eql(...)`. ([@backus][])
 * Add autocorrect support for `BeEql` cop. ([@backus][])
 * Add `MessageExpectation` cop for enforcing consistent style of either `expect(...).to receive` or `allow(...).to receive`. ([@backus][])
+* Add `RSpec::Predicate` cop. ([@haeky][])
```

Comment:
Can you make this just `Predicate`? All of the other cops do not include the `RSpec` prefix in the changelog